### PR TITLE
chore(validation): ⚠️ Fixing Alerts and Validation 📟

### PR DESCRIPTION
**Pull request by S-ayanide**
Signed-off-by: Sayan Mondal <sayan@chaosnative.com>

## Proposed changes

* Workflow and a sub-workflow radio button has to be selected to move forward
* Minimum one experiment has to be selected to move forward after selecting custom workflow

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [X] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [X] I have signed the commit for DCO to be passed.
- [X] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: litmus-portal/frontend/src/views/CreateWorkflow/ChooseWorkflow/index.tsx [hunk 1]
```diff
@@ -108,7 +119,7 @@ const ChooseWorkflow = forwardRef((_, ref) => {
                 templates
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/CreateWorkflow/ChooseWorkflow/index.tsx:119]: translations missing

File: litmus-portal/frontend/src/views/CreateWorkflow/ChooseWorkflow/index.tsx [hunk 2]
```diff
@@ -128,7 +139,7 @@ const ChooseWorkflow = forwardRef((_, ref) => {
                 <strong> My Hubs</strong>
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/CreateWorkflow/ChooseWorkflow/index.tsx:139]: translation missing, use fontWeight if possible

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]
@S-ayanide PTAL minor changes are needed.

**Review by ishangupta-ds — APPROVED** [review]
LGTM

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**